client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/ProxyHelper.java

```diff
@@ -12,7 +12,7 @@
  */
 final class ProxyHelper {
 
-    public static final String WARNING_MSG = "Enabling Proxy server settings on %s is known to change the proxy server settings at the machine Level!\n"
+    public static final String WARNING_MSG = "Enabling Proxy server settings on %s is known to change the proxy server settings at the machine Level!%n"
```

Comment:
@mach6 : Sonar highlighted this with the following reason:

This format string include a newline character (\n). In format strings, it is generally preferable better to use %n, which will produce the platform-specific line separator.